### ISSUE TITLE
Modifying the Contact Us page

### ISSUE THREAD

**Issue by rithvika1818** [BamaCharanChhandogi/Diabetes-Prediction#72]
Hey @BamaCharanChhandogi, I felt that the contact us page can be modified in terms of colors and text. I will add relevant icons for the boxes(name, email, message) and make the page more presentable. Here i am attaching a sample image regarding the icons. 

Please assign this issue to me. I will not surely disappoint.

Thank you in advance!

![contact_us](https://github.com/BamaCharanChhandogi/Diabetes-Prediction/assets/142740556/d37abbf9-bca9-4055-b3cf-0f469bb5d72d)

**Comment by BamaCharanChhandogi** [comment]
Thank you for creating this issue! 🎉 We'll look into it as soon as possible. In the meantime, please make sure to provide all the necessary details and context. If you have any questions or additional information, feel free to add them here. Your contributions are highly appreciated! 😊

You can also check our [CONTRIBUTING.md](https://github.com/BamaCharanChhandogi/Diabetes-Prediction/blob/main/CONTRIBUTING.md) for guidelines on contributing to this project.

**Comment by rithvika1818** [comment]
Yeah sure @BamaCharanChhandogi, I will work on it. Can you please label this as gssoc tag and assign it to me? and also if you are fine, I will add the icons as well. That will make the page more presentable

**Comment by BamaCharanChhandogi** [comment]
Yeah, you can do that.

**Comment by rithvika1818** [comment]
Thanks a lot. I am on it!

**Comment by rithvika1818** [comment]
hey @BamaCharanChhandogi, i am trying to make the contact page functional using smtp server in flask. My approach is to send to admin(your mail id) with the help of credentials hosted on smtp server. But for Configuring SMTP Service On the Server for both gmail and outlook, you need to take help of other sites which are either paid or taking 3-4 days to approve my request. but this can be easily done with javascript as there is a smtp js cdn. Can you suggest anything on this?
